src/main/java/se/liu/ida/hefquin/cli/RunBGPOverNeo4j.java

```diff
@@ -85,6 +70,7 @@ protected RunBGPOverNeo4j( final String[] argv ) {
 		add(argSuppressResultPrintout, "--suppressResultPrintout", "Do not print out the query result");
 
 		addModule(modQuery);
+		addModule(modLPG2RDFConfiguration);
```

Comment:
Better change the order of these two.
```suggestion
		addModule(modLPG2RDFConfiguration);
		addModule(modQuery);
```